src/main/java/auto_batcher/Student.java

```diff
@@ -24,5 +24,19 @@ public Student(FirstLastName name, Long year){
 		this.year = year;
 	}
 	
+	public String getFirstName(){
```

Comment:
this class has not been implemented correctly.
as i note above `FirstLastName` should be renamed to `NameTuple.`

then all this class needs to do is extend NameTuple.


``` java
public class Student extends NameTuple
{

       public Student (string firstName, lastName, year) {
             super(firstName,LastName);
             this.year = year.
       }
	private Long year;
	       
	public Long getYear(){
		return year;
	}

}
```